src/styles/colors.ts

```diff
@@ -0,0 +1,97 @@
+import {Color} from './themes/ThemeColors';
+
+/**
+ * DO NOT import colors.js into files. Use ../themes/default.js instead.
+ */
+const colors: Record<string, Color> = {
```

Comment:
I think of a safer option is to define colors:
```ts
const colors = {
   ...
} as const;

// OR

const colors = {
   ...
} satisfies Record<string, Color>;
```

This way we will get intellisense when accessing `colors` (VSCode can prompt with color names when typing `colors.`). 

⚠️ On main branch there is still old version of Typescript (4.8.4), `satisfies` keyword [was introduced in 4.9](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html). So if you want to use it, you'd have to rebase this branch to [typescript/onyx-types](https://github.com/Expensify/App/pull/24054) where TS version is bumped. The problem is that the PR is blocked for now and I don't know when exactly we will be able to merge it.

Reply:
Agree, this is a better alternative 👍 Changed!